src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,10 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        return false;
+        StringBuilder clean = new StringBuilder(text.replaceAll("[^a-zA-Z0-9]", "").toLowerCase());
+        String cleanStr = clean.toString();
```

Comment:
```suggestion
        String cleanText = new StringBuilder(text.replaceAll("[^a-zA-Z0-9]", "").toLowerCase()).toString();
```

Reply:
You suggest to make cleanText as String, but we will need StringBuilder to use reverse() method in your next suggestion. Am I right?

Reply:
but rename the variable